array-methods/foreach/src/__tests__/1_forEach_exercises.test.js

```diff
@@ -92,7 +98,15 @@ describe("Exercise 2", () => {
 `;
 
 export const createLibraVariations = (array, callback) => {
-  // Your solution here
+  let libraGroup;
+  array.forEach((user, index) => {
+    if (index < array.length / 2) {
+      libraGroup = "A";
+    } else {
+      libraGroup = "B";
+    }
+    callback({ email: user.email, libraGroup: libraGroup });
```

Comment:
Could use ES6 shorthand here to cut them keystrokes 
```suggestion
    callback({ email: user.email, libraGroup });
```